package_scripts/start_www.sh

```diff
@@ -2,26 +2,8 @@
 
 set -euo pipefail
 
-source package_scripts/common.sh
-
 THEMES_PATH=$(pwd)
-# re-export with some fallbacks
-export SEARCH_API_URL=${SEARCH_API_URL:-"https://discussions-rc.odl.mit.edu/api/v0/search/"}
-export OCW_STUDIO_BASE_URL=${OCW_STUDIO_BASE_URL:-"http://ocw-studio-rc.odl.mit.edu/"}
-export WWW_HUGO_CONFIG_PATH=${WWW_HUGO_CONFIG_PATH:-"https://live-qa.ocw.mit.edu/"}
-export WWW_CONTENT_PATH=${WWW_CONTENT_PATH:-"https://live-qa.ocw.mit.edu/"}
-
 
-if [[ -z "${WWW_HUGO_CONFIG_PATH}" ]]; then
-  # Path to Hugo config not set, fetch default from Github
-  clone_or_pull_repo git@github.com:mitodl/ocw-hugo-projects.git private/ocw-hugo-projects
-  WWW_HUGO_CONFIG_PATH=$THEMES_PATH/private/ocw-hugo-projects/ocw-www/config.yaml
-fi
-if [[ -z "${WWW_CONTENT_PATH}" ]]; then
-  # Path to www content not set, fetch default from ocw-content-rc
-  clone_or_pull_repo git@github.mit.edu:ocw-content-rc/ocw-www.git private/ocw-www
-  WWW_CONTENT_PATH=$THEMES_PATH/private/ocw-www
-fi
```

Comment:
@gumaerc I removed these lines (and similar lines in other scripts) in response to your comment

> I noted this below regarding WWW_CONTENT_PATH but there are some defaults assigned in the various bash scripts that handle starting courses or [www](http://www/). They seem to be getting in the way of envalid showing the error messages you referred to in the testing instructions. If we're setting defaults using envalid then we should remove the defaults from the bash scripts.

I'm not 100% sure if you were suggesting I remove this git-cloning stuff, or that I only remove the simple defaults above on lines 9–12.

But... I think we should get rid of this "fallback to git cloning". It's basically setting a default for `WWW_CONTENT_PATH`. It complicates stuff, and it's not mentioned in the readme. 

What I'd really like to do (in a separate PR) is use a commander script (like `with-env.ts` is) to along the lines of
```ts
import { program } from "commander"

const start = program
  .description("Start ocw-hugo-themes development servers.")

start
  .command("course")
  .description("Start webpack+hugo dev servers using course theme.")
  .action(() => {
    // Check content directory exists, give useful error if not
    // possibly offer option to clone the repo.
    console.log('TODO: command to actually start course.')
  })

start
  .command("www")
  .description("Start webpack+hugo dev servers using www theme.")
  .action(() => {
    console.log('TODO: command to actually start www.')
  })

start.parse()
```
then `yarn start` would give:
```
➜ yarn start
Usage: start [options] [command]

Start ocw-hugo-themes development servers.

Options:
  -h, --help      display help for command

Commands:
  course          Start webpack+hugo dev servers using course theme.
  www             Start webpack+hugo dev servers using www theme.
  help [command]  display help for command
```

Reply:
Yea, I think this is fine to do. These defaults and the automatic cloning were added as part of the last refactor of the local dev experience. Alice and I had the goal of "you shouldn't have to configure anything to get up and running and start working on the theme." Obviously your local terminal would still need to be set up with git credentials to MIT Github Enterprise, but beyond that running `npm run start:course` after that PR without any env would give you a usable site. This is the PR that stuff was added in: https://github.com/mitodl/ocw-hugo-themes/pull/231. Your work here aims to accomplish the same thing in a more sane manner. You should be able to spin up a course / `ocw-www` without having to manually go out and clone anything, but it doesn't necessarily need to be handled here and could be done in another PR.